src/renderer/dx/DxRenderer.cpp

```diff
@@ -2040,26 +2016,12 @@ float DxEngine::GetScaling() const noexcept
 // - iDpi - <unused>
 // Return Value:
 // - S_OK
-[[nodiscard]] HRESULT DxEngine::GetProposedFont(const FontInfoDesired& pfiFontInfoDesired,
-                                                FontInfo& pfiFontInfo,
-                                                int const iDpi) noexcept
-{
-    Microsoft::WRL::ComPtr<IDWriteTextFormat> format;
-    Microsoft::WRL::ComPtr<IDWriteTextFormat> formatItalic;
-    Microsoft::WRL::ComPtr<IDWriteTextAnalyzer1> analyzer;
-    Microsoft::WRL::ComPtr<IDWriteFontFace1> face;
-    Microsoft::WRL::ComPtr<IDWriteFontFace1> faceItalic;
-    LineMetrics lineMetrics;
-
-    return _GetProposedFont(pfiFontInfoDesired,
-                            pfiFontInfo,
-                            iDpi,
-                            format,
-                            formatItalic,
-                            analyzer,
-                            face,
-                            faceItalic,
-                            lineMetrics);
+[[nodiscard]] HRESULT DxEngine::GetProposedFont(const FontInfoDesired& /*pfiFontInfoDesired*/,
+                                                FontInfo& /*pfiFontInfo*/,
+                                                int const /*iDpi*/) noexcept
+{
+    // No-op

```

Comment:
There's no `_GetProposedFont` in this class anymore. And this method is not really used by anyone. So I just nuked this.

Reply:
Please leave it hooked up if you can. It's not technically used at this exact moment, but this renderer does technically still work in conhost.exe if we wanted it to. 

Or if it's too hard to hook up, at least make it return an error like E_NOTIMPL not S_OK.

Reply:
(I'm going to stay on blocking until this one is worked out at least.)

Reply:
I've updated the PR. Hope i'm doing it correctly.